gcp/observability/observability.go

```diff
@@ -80,4 +85,18 @@ func Start(ctx context.Context) error {
 // Note: this method should only be invoked once.
 func End() {
 	defaultLogger.Close()
+	if exporter != nil {
```

Comment:
Let's create a `stopOpenCensus` function that does the cleanup needed due to `startOpenCensus`, and put it right next to it so we can be sure everything pairs up between start/end.

Reply:
Done. Thanks.